### PR TITLE
WIP: Burial and pension intro page content

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3610
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3620
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3611

Still waiting on some tweaks.

screencapture-localhost-3001-burials-and-memorials-application-530-introduction-1498855726775
screencapture-localhost-3001-pension-application-527ez-introduction-1498855697357

### CODE REVIEW

**Review by cehsu — APPROVED** [review]

File: src/js/burials/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -17,16 +17,45 @@ class IntroductionPage extends React.Component {
...
+            {...this.props.saveInProgress}/>
+        <div className="process schemaform-process schemaform-process-sip">
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>What you need to fill out this application</h6></div>
+              <ul>
+                <li>Veteran’s Social Security number or VA file number (required)</li>
+                <li>Veteran’s military discharge papers (DD214 or other separation documents)</li>
+                <li>Veteran’s death certificate</li>
```
Comment (cehsu) [inline src/js/burials/components/IntroductionPage.jsx]: for death certificate @J-Quag [suggested ](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3610#issuecomment-312363978)indicating "If claiming service connected death"

File: src/js/burials/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -17,16 +17,45 @@ class IntroductionPage extends React.Component {
...
+                <li>Receipts for transportation costs</li>
```
Comment (cehsu) [inline src/js/burials/components/IntroductionPage.jsx]: for transportation receipts @J-Quag for death certificate @J-Quag suggested indicating "If claiming transportation costs of the Veteran's remains"

File: src/js/pensions/components/IntroductionPage.jsx [hunk 3]
```diff
@@ -17,16 +17,49 @@ class IntroductionPage extends React.Component {
...
+              <div><h5>Apply for Benefits</h5></div>
+              <p>Complete this form.</p>
+            </li>
+            <li className="process-step list-three">
+              <div><h5>VA Review</h5></div>
+              <div><h6>What if VA needs more information?</h6></div>
+              <ul><li>We will contact you if we need more information.</li></ul>
+              <div><h6>How long does it take VA to make a decision?</h6></div>
+              <ul><li>We process claims in the order we receive them, unless a claim requires priority processing.</li></ul>
+            </li>
```
Comment (cehsu) [inline src/js/pensions/components/IntroductionPage.jsx]: @J-Quag [suggested](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3611#issuecomment-312365845) potentially mentioning a two-week wait period for following up, is this the appropriate place for that content if the decision is made to include it?

File: src/js/burials/components/IntroductionPage.jsx [hunk 4]
```diff
@@ -17,16 +17,45 @@ class IntroductionPage extends React.Component {
...
+            {...this.props.saveInProgressActions}
+            {...this.props.saveInProgress}/>
+        <div className="process schemaform-process schemaform-process-sip">
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>What you need to fill out this application</h6></div>
+              <ul>
+                <li>Veteran’s Social Security number or VA file number (required)</li>
+                <li>Veteran’s military discharge papers (DD214 or other separation documents)</li>
```
Comment (cehsu) [inline src/js/burials/components/IntroductionPage.jsx]: Does the fact that military history is [not required](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3610#issuecomment-312363978) for this form mean that it should not be listed here? Or that it should not have the word required after it?

File: src/js/burials/components/IntroductionPage.jsx [hunk 5]
```diff
@@ -17,16 +17,45 @@ class IntroductionPage extends React.Component {
...
-        </p>
-        <p>
-          VA uses the information you submit to determine your eligibility and to provide you with the best service.
-        </p>
-        <p>
-          Federal law provides criminal penalties, including a fine and/or imprisonment for up to 5 years, for concealing a material fact or making a materially false statement. (See <a href="https://www.justice.gov/usam/criminal-resource-manual-903-false-statements-concealment-18-usc-1001" target="_blank">18 U.S.C. 1001</a>)
-        </p>
+        <FormTitle title="Apply for burial benefits"/>
+        <p>This application is equivalent to Form 21P-530 (Application for Burial Benefits).</p>
+        <SaveInProgressIntro
```
Comment (cehsu) [inline src/js/burials/components/IntroductionPage.jsx]: This will be merged after SIP?

File: src/js/burials/components/IntroductionPage.jsx [hunk 6]
```diff
@@ -17,16 +17,45 @@ class IntroductionPage extends React.Component {
...
+              <ul>
+                <li>Veteran’s Social Security number or VA file number (required)</li>
+                <li>Veteran’s military discharge papers (DD214 or other separation documents)</li>
+                <li>Veteran’s death certificate</li>
+                <li>Receipts for transportation costs</li>
+              </ul>
+              <p><a href="http://www.va.gov/ogc/apps/accreditation/index.asp">An accredited representative</a> with a Veterans Service Organization (VSO) can help you pick the right program.</p>
+              <h6>Learn about other survivor and dependent benefits</h6>
+              <ul>
+                <li>If you’re the survivor or dependent of a Veteran who died in the line of duty or from a service-related illness, you may be able to get a benefit called Dependency and Indemnity Compensation. <a href="/burials-and-memorials/survivor-and-dependent-benefits/compensation/">See if you can get compensation</a>.</li>
```
Comment (cehsu) [inline src/js/burials/components/IntroductionPage.jsx]: Should we use language consistent with that used elsewhere? Such as "Learn more about Dependency and Indemnity Compensation"?

File: src/js/common/schemaform/FormStartControls.jsx [hunk 7]
```diff
@@ -55,11 +55,11 @@ class FormStartControls extends React.Component {
               onButtonClick={this.handleLoadForm}
               buttonText="Resume previous application"
               buttonClass="usa-button-primary"/>
-          <ProgressButton
+          {!this.props.resumeOnly && <ProgressButton
               onButtonClick={this.toggleModal}
               buttonText="Start over"
```
Comment (cehsu) [inline src/js/common/schemaform/FormStartControls.jsx]: I wonder if the start over button should be on the left and the resume previous application button should be on the right with the chevrons?

File: src/js/pensions/components/IntroductionPage.jsx [hunk 8]
```diff
@@ -17,16 +17,49 @@ class IntroductionPage extends React.Component {
...
+            resumeOnly
+            {...this.props.saveInProgressActions}
+            {...this.props.saveInProgress}/>
+        <div className="process schemaform-process schemaform-process-sip">
+          <ol>
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>What you need to fill out this application</h6></div>
+              <ul>
+                <li>Social Security number (required)</li>
```
Comment (cehsu) [inline src/js/pensions/components/IntroductionPage.jsx]: Should we also say: "Social Security number or VA file number (required)" here?

File: src/js/burials/components/IntroductionPage.jsx [hunk 9]
```diff
@@ -17,16 +17,45 @@ class IntroductionPage extends React.Component {
...
+            <li className="process-step list-one">
+              <div><h5>Prepare</h5></div>
+              <div><h6>What you need to fill out this application</h6></div>
+              <ul>
+                <li>Veteran’s Social Security number or VA file number (required)</li>
+                <li>Veteran’s military discharge papers (DD214 or other separation documents)</li>
+                <li>Veteran’s death certificate</li>
+                <li>Receipts for transportation costs</li>
+              </ul>
+              <p><a href="http://www.va.gov/ogc/apps/accreditation/index.asp">An accredited representative</a> with a Veterans Service Organization (VSO) can help you pick the right program.</p>
```
Comment (cehsu) [inline src/js/burials/components/IntroductionPage.jsx]: It [looks](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3610#issuecomment-312041975) like this should be: "fill out the claim"

File: src/js/pensions/components/IntroductionPage.jsx [hunk 10]
```diff
@@ -17,16 +17,49 @@ class IntroductionPage extends React.Component {
...
+                <li>Military history (required)</li>
+                <li>Your financial information and the financial information of your dependents (required)</li>
+                <li>Work history</li>
+                <li>Bank account direct deposit information</li>
+                <li>Medical information</li>
+              </ul>
+              <p><a href="http://www.va.gov/ogc/apps/accreditation/index.asp">An accredited representative</a> with a Veterans Service Organization (VSO) can help you pick the right program.</p>
```
Comment (cehsu) [inline src/js/pensions/components/IntroductionPage.jsx]: It [looks](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3611#issuecomment-312110401) like this should be "fill out the claim".

File: src/js/pensions/components/IntroductionPage.jsx [hunk 11]
```diff
@@ -17,16 +17,49 @@ class IntroductionPage extends React.Component {
...
+            <li className="process-step list-two">
+              <div><h5>Apply for Benefits</h5></div>
+              <p>Complete this form.</p>
+            </li>
+            <li className="process-step list-three">
+              <div><h5>VA Review</h5></div>
+              <div><h6>What if VA needs more information?</h6></div>
+              <ul><li>We will contact you if we need more information.</li></ul>
+              <div><h6>How long does it take VA to make a decision?</h6></div>
+              <ul><li>We process claims in the order we receive them, unless a claim requires priority processing.</li></ul>
```
Comment (cehsu) [inline src/js/pensions/components/IntroductionPage.jsx]: I wonder whether @J-Quag's [comment](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3611#issuecomment-312365845) regarding dates was in reference to this?

File: src/js/burials/components/IntroductionPage.jsx [hunk 12]
```diff
@@ -17,16 +17,45 @@ class IntroductionPage extends React.Component {
...
+              <p>Complete this form.</p>
+            </li>
+            <li className="process-step list-three">
+              <div><h5>VA Review</h5></div>
+              <div><h6>What if VA needs more information?</h6></div>
+              <ul><li>We will contact you if we need more information.</li></ul>
+            </li>
+            <li className="process-step list-four">
+              <div><h5>Decision</h5></div>
+              <ul><li>You’ll get a notice in the mail if your claim was approved.</li></ul>
```
Comment (cvalarida) [inline src/js/burials/components/IntroductionPage.jsx]: And if not?

File: src/js/pensions/components/IntroductionPage.jsx [hunk 13]
```diff
@@ -17,16 +17,49 @@ class IntroductionPage extends React.Component {
...
+            </li>
+            <li className="process-step list-four">
+              <div><h5>Decision</h5></div>
+              <ul><li>If we need more information from you with the processing of your claim you will be notified by a letter. At the end of the process you will be notified via mail of the decision.</li></ul>
```
Comment (cehsu) [inline src/js/pensions/components/IntroductionPage.jsx]: Is it redundant to mention this again, as it is already mentioned on line 53?

**Review by cvalarida — APPROVED** [review]

File: src/js/burials/components/IntroductionPage.jsx (hunk 12 above)
Comment (cehsu) [inline src/js/burials/components/IntroductionPage.jsx]: Hm, good point: cc @peggygannon @stacyhlinn @J-Quag?

File: src/js/burials/components/IntroductionPage.jsx (hunk 5 above)
Comment (jbalboni) [inline src/js/burials/components/IntroductionPage.jsx]: That's been the plan.

**Comment by jbalboni** [comment]
@cehsu I'm waiting to address some of your comments until Peggy has a chance to adjust her copy based on Josh's feedback.

**Comment by jbalboni** [comment]
I'm going to merge this, even though it seems like the content/format may change again today or tomorrow, so we have a more realistic version of what the intro pages look like for demos tomorrow.

**Pull request merged**